components/automate-gateway/integration/license_usage_nodes_test.go

```diff
@@ -110,7 +110,7 @@ func (suite *GatewayTestSuite) TestLicenseUsageNodes() {
 		Name:          "my job for aws-api node manager",
 		Tags:          []*common.Kv{},
 		Type:          "exec",
-		Profiles:      []string{"https://github.com/vjeffrey/try-inspec-aws-profile/archive/master.tar.gz"},
+		Profiles:      []string{"https://github.com/chef/automate/raw/master/components/compliance-service/test_data/inspec_profiles/test-aws-profile-2.0.0.tar.gz"},
```

Comment:
I guess we'd be a little safer if this wasn't master, but a commit ref 🤔

Reply:
I agree, will make the change tomorrow, thank you!